bbc/digital-paper-edit-firebase
# wip: prevent user from leaving the page if transcript is still uploading

**Is your Pull Request request related to another [https://github.com/bbc/digital-paper-edit-storybook/issues/35](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      


**Describe what the PR does**    
- alerting the user and preventing from leaving/reloading the page if the transcript is still uploading



**Additional context**    

The issue is linked to digital-paper-edit-storybook repo, but the logic is implemented here.

**To test**
- create new Transcript and upload a video
- while it's still uploading try to reload/close the browser page
- it should alert that ''Changes that you made may not be saved" if you leave the page

Ps: alert message is not customisible -  It is widely considered a security issue, and most browsers have removed support.
